docs/installation.rst

```diff
@@ -1,86 +1,83 @@
+************
 Installation
-============
+************
 
-Installing composer is quick and simple.
+Installing via composer is quick and simple, and the plugin is `available on Packagist <https://packagist.org/packages/friendsofcake/crud>`_.
```

Comment:
Is it worth moving the Packagist link to under the 'Using composer' sub-section then? I think it's worth keeping a link somewhere, as it's easier to browse releases and check dependencies on Packagist.

I'll move it into that section for the meantime, pending further discussion.